pages/task/read/views/pil.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Fragment } from 'react';
+import moment from 'moment';
+import { Snippet } from '@asl/components';
+import { dateFormat, procedureDefinitions } from '../../../../constants';
+import { connect } from 'react-redux';
+
+const Pil = ({ profile }) => {
+  const pil = profile.pil;
+  return (
+    <div className="govuk-grid-row">
+      <div className="govuk-grid-column-three-quarters" id="training">
```

Comment:
I don't think we should have the grid containers in here. The component should _only_ define itself and not the context in which it exists.